src/MUMPSfuncs.jl

```diff
@@ -116,6 +116,19 @@ function applyMUMPS!(factor::MUMPSfactorization{Float64},rhs::SparseMatrixCSC{Fl
 	return x
 end
 
+function applyMUMPS!(factor::MUMPSfactorization{Float64},rhs::SparseVector{Float64}, x::Array{Float64,1},tr::Int=0)
+	nrhs = 1
+	nzrhs = nnz(rhs)
+	colptr = [1;3]
```

Comment:
Should this `colptr` match the `colptr` for a `SparseMatrixCSC` representation of `rhs::SparseVector`? If so, shouldn't `colptr = [1, nzrhs+1]`? If not, kindly ignore :).